docs/zephyr-integration.md

```diff
@@ -0,0 +1,454 @@
+# Zephyr Test Management Integration
+
+## Overview
+
+The Zephyr Test Management integration provides AI assistants with comprehensive test management capabilities through the Model Context Protocol (MCP). This integration enables seamless interaction with Zephyr Cloud for test case creation, test planning, execution tracking, and issue coverage analysis directly through conversational AI interfaces.
+
+### Key Benefits
+
+- **AI-Assisted Test Management**: Leverage conversational AI for creating and managing test cases, plans, and executions
+- **Issue Traceability**: Analyze test coverage for JIRA issues and requirements
+- **Automated Workflows**: Streamline test planning and execution processes
+- **Comprehensive API Coverage**: Full access to Zephyr Cloud functionality through standardized MCP tools
+
+## Setup
+
+### Environment Variables
+
+The Zephyr integration requires the following environment variables:
+
+#### Required
+- `ZEPHYR_ACCESS_TOKEN` (required): JWT access token for Zephyr Cloud API authentication
+  - Obtain from Zephyr Cloud account settings
+  - Format: Standard JWT token (e.g., `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...`)
+
+#### Optional
+- `ZEPHYR_PROJECT_KEY` (optional): Default project scope for operations
+  - Format: Uppercase project key (e.g., `PROJ`, `DEV`, `TEST`)
+  - When specified, operations will default to this project unless overridden
+
+- `ZEPHYR_BASE_URL` (optional): Custom API endpoint for enterprise deployments
```

Comment:
are we supporting also zephyr essentials or what is the reason of letting the user sets a different base_url?